File: applications/account-gemfire-rabbit-stream-sink/src/main/java/com/vmare/account/gemfire/rabbit/stream/sink/repository/AccountRepository.java
package com.vmare.account.gemfire.rabbit.stream.sink.repository;

import com.vmware.tanzu.data.services.rabbitmq.streaming.account.domain.Account;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface AccountRepository extends CrudRepository<Account,String> {
}

File: applications/account-jdbc-amqp-sink/src/main/kotlin/com/vmware/tanzu/data/services/rabbitmq/streaming/account/repository/AccountRepository.kt
package com.vmware.tanzu.data.services.rabbitmq.streaming.account.repository

import com.vmware.tanzu.data.services.rabbitmq.streaming.account.domain.Account
import com.vmware.tanzu.data.services.rabbitmq.streaming.account.domain.AccountEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface AccountRepository : CrudRepository<AccountEntity,String> {
}